controllers/secretagentconfiguration_controller.go

```diff
@@ -82,48 +83,48 @@ func (reconciler *SecretAgentConfigurationReconciler) Reconcile(req ctrl.Request
 	// TODO remove this and move setup of secret manager to a container
 	// add conatiner to GenKeyConfig
 
-	secretMgr := secretmanager.NewSecretManager(rclient, instance.AppConfig)
-
-	//if instance.Spec.AppConfig.SecretsManager != v1alpha1.SecretsManagerNone &&
-	//	instance.Spec.AppConfig.CredentialsSecretName != "" {
-	//	var dir string
-	//	var cloudCredNS string
-
-	//	if len(reconciler.CloudSecretsNamespace) > 0 {
-	//		cloudCredNS = reconciler.CloudSecretsNamespace
-	//	} else {
-	//		cloudCredNS = instance.Namespace
-	//	}
-
-	//	// load credentials secret
-	//	secObject, err := k8ssecrets.LoadSecret(reconciler.Client,
-	//		instance.Spec.AppConfig.CredentialsSecretName, cloudCredNS)
-	//	if err != nil {
-	//		log.Error(err, "error loading cloud credentials secret from the Kubernetes API",
-	//			"secret_name", instance.Spec.AppConfig.CredentialsSecretName,
-	//			"namespace", cloudCredNS)
-	//		return ctrl.Result{}, err
-	//	}
-	//	// Create temporary dir for gcp credentials if needed
-	//	if instance.Spec.AppConfig.SecretsManager == v1alpha1.SecretsManagerGCP {
-	//		dir, err = ioutil.TempDir("", "cloud_credentials-*")
-	//		if err != nil {
-	//			log.Error(err, "couldn't create a temporary credentials dir")
-	//			return ctrl.Result{}, err
-	//		}
-	//		// clean up after ourselves
-	//		defer os.RemoveAll(dir)
-	//	}
-
-	//	// load cloud credentials to envs and/or files
-	//	if err := manageCloudCredentials(instance.Spec.AppConfig.SecretsManager, secObject, dir); err != nil {
-	//		log.Error(err, "error loading cloud credentials from secret provided",
-	//			"secret_name", instance.Spec.AppConfig.CredentialsSecretName,
-	//			"namespace", cloudCredNS)
-	//		return ctrl.Result{}, err
-	//	}
-
-	//}
+	//	secretMgr := secretmanager.NewSecretManager(rclient, instance.AppConfig)
+
+	if instance.Spec.AppConfig.SecretsManager != v1alpha1.SecretsManagerNone &&
```

Comment:
worth moving all this to a function?

Reply:
https://github.com/ForgeRock/secret-agent/pull/159/commits/c6c73a70cb938aaaae4790069eb64e480762ebb4 I have it started in the package wasn't going to touch this section at the moment.